main/res/values/strings.xml

```diff
@@ -252,6 +253,8 @@
     <string name="info_stored_image">New image saved to:</string>
     <string name="info_storing_static_maps">Trying to store static maps</string>
     <string name="info_cache_saved">The cache has been stored locally</string>
+    <string name="info_calculator_state_saved">Current state of the calculator has been saved to waypoint.</string>
```

Comment:
@SammysHP  @S-Bartfast 
I understand all pro and con of this topic. I think I need to see this "in action" and then suggest a change if needed. 
So my proposal would be to leave it as it is for now and review after merge to `master` when testing the implementation on a real day usage.